packages/wasm/rollup.config.js

```diff
@@ -0,0 +1,15 @@
+const pkg = require('./package.json');
+
+const dependencies = Object.keys(pkg.dependencies || {});
+
+export default [
+  {
+    input: 'src/index.js',
+    output: { exports: 'named', file: 'dist/index.js', format: 'cjs' },
+    external: ['path'].concat(dependencies)
```

Comment:
This is missing `'fs'` and produces a warning:

```
src/index.js → dist/index.js...
(!) Unresolved dependencies
https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency
fs (imported by src\index.js)
created dist/index.js in 35ms
```

Maybe could do all builtin modules?

```js
const pkg = require('./package.json')
const { builtinModules } = require('module')
const dependencies = Object.keys(pkg.dependencies || {})

export default {
    external: [ ...builtinModules, ...dependencies ]
}
```

Reply:
@jamen please feel free to commit to the branch. as a member of Plugin Maintainers, you have permissions.

Reply:
Thank you, done.